resources/js/Pages/Countries.vue

```diff
@@ -0,0 +1,50 @@
+<script setup>
+import Country from '../Components/Country.vue'
+import { useForm } from '@inertiajs/vue3'
+
+function storeCountry() {
+  storeCountryForm.post('/countries')
+  storeCountryForm.reset()
```

Comment:
You might use the **onSuccess** callback here to reset form inputs to their original state.